01-tree-specification.bs

```diff
@@ -118,21 +117,15 @@ In one tree, completeness MUST be guaranteed, unless indicated otherwise (as is
 
 # Initialization # {#init}
 
-A client SHOULD be initiated using a URL.
-The client MUST dereference the URL, which will result in a set of [[!rdf-concepts]] triples or quads.
-When the URL after all redirects, is used in a triple `?c tree:view <> .`, a client MUST assume the URL after redirects is an identifier of the intended root node of the collection in `?c`.
+The main TREE specification specifies a TREE-based client MUST be initiated either by using the IRI of the `tree:Collection`, or by using the IRI of the root `tree:Node`.
+The client MUST start by dereferencing the IRI resulting in a set of [[!rdf-concepts]] triples or quads:
 
-Note: Dereferencing in this specification also means parsing the RDF triples or quads from the HTTP response. TREE does not limit the content-types that can be used to represent RDF triples. Client developers should do a best-effort for their community.
+ 1. Detecting and handling a `tree:Collection`: If exactly one `<collectionIRI> tree:view ?o` triple pattern can be matched, its object identifies the root `tree:Node`. If the current page IRI is not equal to that node’s IRI, the client MUST dereference the IRI of the root node and restart the initialization process from there. This behavior ensures backward compatibility and aligns with legacy practices. More advanced discovery mechanisms (e.g., supporting multiple views or selection logic) are out of scope for this specification and will be addressed by the report on Discovery and Context Information.
+ 2. Detecting and handling a root `tree:Node`: In this case, the final IRI after all HTTP redirects MUST correspond to the object of a `tree:view` triple linking it to a `tree:Collection`. This allows the client to verify that it has reached a valid entry point into the TREE structure.
```

Comment:
These HTTP redirects still make sense to me, but if read it correctly, this is actually not allowed anymore with the change above: "client MUST be initiated either by using the IRI of the `tree:Collection`, or by using the IRI of the root `tree:Node`." as it is the IRI after redirect which will correspond with the root `tree:Node`, but in the scenario of HTTP redirects, we assume the client was initiated with a IRI before redirects not being the IRI of the root `tree:Node`, neither it is the IRI of the `tree:Collection` in this scenario. Right?